backend/common/utils/leaderboard.py

```
@@ -0,0 +1,73 @@
+    f = open(path, "w")
+    writer = csv.writer(f)
+    writer.writerow(["rank", "user_id", "name", "score"])
+    for entry in ranked:
+        try:
+            writer.writerow([
+                entry["rank"],
+                entry["user_id"],
+                entry["name"],
+                round(entry["score"], 2),
+            ])
+        except:
```

Ruff (0.15.17)

[error] Do not use bare `except`

(E722)